strangerstudios/paid-memberships-pro
# Adding SVG icon to PMPro block category.

### All Submissions:

* [ x Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?





### Changes proposed in this Pull Request:

Adding block category icon.

<img width="387" alt="Screen Shot 2020-04-24 at 8 01 43 AM" src="https://user-images.githubusercontent.com/636521/80215353-eac3a100-8601-11ea-9c46-dc669b196b90.png">


### How to test the changes in this Pull Request:

1. Add a block on a post or page by clicking the + icon on the top left
2. Observe the new category icon.
3. Ensure no breaking JS errors

### Other information:




### Changelog entry

> Adding block category icon.
